packages/layout/__tests__/layout-test.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright IBM Corp. 2018, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { createSassRenderer, convert } from '@carbon/test-utils/scss';
+import { spacing, layout, fluidSpacing, container, iconSize } from '../src';
+
+function formatStep(name, index) {
+  let step = '' + index;
```

Comment:
```suggestion
  let step = index;
```

couldn't we just set this as `index`? then comparison on the next line would be between two numbers and then either L14 or L16 would turn `step` into a string anyway